git-hooks/pre-commit-in-container

```diff
@@ -250,6 +250,6 @@ if $go_apache_check_failed || $go_copyright_check_failed || \
    $junit_hook_check_failed || $stale_file_in_skip_files; then
   exit 1
 fi
-
+exec ./git-hooks/pre-commit-gpl-apache
 # Remove the trap handler.
 trap "" EXIT
```

Comment:
This line is now not hit because `exec` doesn't return.

Feels like a bit of a code smell to have one script with lots of low-level function call another similar script at the end.  I think you should either add the check to this file, even though it's getting a bit big, or, refactor so there's a short script in pre-commit-in-container that invokes other scripts to do well-defined pieces of work.